### PR TITLE
Bump domjudge version to 8.1.3

### PR DESCRIPTION
**Pull request by qinmaye**

### CODE REVIEW

File: domserver.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ sudo apt-get update
 
 ```shell
 sudo apt install acl zip unzip mariadb-server apache2 \
-        php php-gd php-cli php-intl php-mbstring php-mysql \
+        php php-fpm php-gd php-cli php-intl php-mbstring php-mysql \
```
Comment (cubercsl) [inline domserver.md]: 是否和上面说的用 apache2 + mod_php 矛盾？

Comment (cubercsl) [reply domserver.md]: https://github.com/DOMjudge/domjudge/commit/9fb512e7aa1e28f736c76f9c3a5f99a723caaa28 FYI

Comment (yang-er) [reply domserver.md]: 为什么apache2 + php-fpm，用nginx不香吗.jpg

Comment (qinmaye) [reply domserver.md]: Update according CSL's comments, please help review again.

**Review by cubercsl — APPROVED** [review]

**Review by yang-er — APPROVED** [review]

**Pull request merged**